### PR TITLE
Automation changes

### PR DESCRIPTION
**Pull request by jayesh05071993**

### CODE REVIEW

File: NHS.Automation/FrameworkPython/configuration_files/ElementLocators.cnf [hunk 1]
```diff
@@ -6,8 +6,10 @@ Label_Hello_Admin = //*[@id='myhscic']
 
 [Home_Page]
 drop_down_click = //*[@id='trust_input']
-graph_label = //*[@id='region_nd_other']
+graph_label = //*[@id='region_display']
+#graph_label = //*[@id='region_nd_other']
```
Comment (Akshay14469) [inline NHS.Automation/FrameworkPython/configuration_files/ElementLocators.cnf]: Remove this

File: NHS.Automation/FrameworkPython/configuration_files/ElementLocators.cnf [hunk 2]
```diff
@@ -6,8 +6,10 @@ Label_Hello_Admin = //*[@id='myhscic']
...
 image_region_vs_peer = //*[@id="chart3"]/a/img
-image_trust_vs_peer = //*[@id="chart2"]/a/img
+image_trust_vs_peer = //*[@id="dvSidePanelTvP"]/a/img
+#image_trust_vs_peer = //*[@id="chart2"]/a/img
```
Comment (Akshay14469) [inline NHS.Automation/FrameworkPython/configuration_files/ElementLocators.cnf]: remove this.

File: NHS.Automation/FrameworkPython/configuration_files/TestConfigurations.cnf [hunk 3]
```diff
@@ -1,7 +1,17 @@
...
+# Application_URL = http://35.176.37.177:8085/NHSD_POC/
+
+Exe_Path = ..\\chromedriver.exe
+# Windows Path will be removed once tested on server
 Exe_Path = /var/www/Driver/chromedriver
-Excel_Path = /var/lib/jenkins/workspace/NHSD_POC/NHS.Automation/FrameworkPython/test_data/TestData.xlsx
+
+# Windows Path will be removed once tested on server
+#Excel_Path = ../test_data/TestData.xlsx
+Excel_Path = /var/www/Automation_TestCases/NHSD_POC/NHS.Automation/FrameworkPython/test_data/TestData.xlsx
```
Comment (Akshay14469) [inline NHS.Automation/FrameworkPython/configuration_files/TestConfigurations.cnf]: Keep this -> Excel_Path = /var/lib/jenkins/workspace/NHSD_POC/NHS.Automation/FrameworkPython/test_data/TestData.xlsx

**Review by Akshay14469 — COMMENTED** [review]
Please do the changes suggested.

**Review by Akshay14469 — DISMISSED** [review]
Looks good to me

**Review by prabhat14176 — DISMISSED** [review]
Looks good to me

File: NHS.Automation/FrameworkPython/configuration_files/TestConfigurations.cnf [hunk 4]
```diff
@@ -1,7 +1,19 @@
 [ConfigDetails]
 Test_Browser = Chrome
+
 Application_URL = http://10.10.1.12:8085/NHSD_POC/
+# Windows Path will be removed once tested on server
+# Application_URL = http://35.176.37.177:8085/NHSD_POC/
+
+API_URL = http://35.176.37.177:8084/search_trust/
```
Comment (Akshay14469) [inline NHS.Automation/FrameworkPython/configuration_files/TestConfigurations.cnf]: Change this to -> API_URL = http://10.10.1.12:8084/search_trust/

File: NHS.Automation/FrameworkPython/common_files/common_functions.py [hunk 5]
```diff
@@ -48,6 +51,28 @@ def login_application(login_cred,password):
     obj_login_class.btn_clk_login()
     obj_login_class.validate_home_page(login_cred)
 
+def validate_trust_data(str_trust_name,value_e1,value_e2,value_e3,value_e4):
+    str_url = read_config.read_config_data('ConfigDetails', 'Application_URL')
+    api_url = read_config.read_config_data('ConfigDetails', 'API_URL')
+
+    str_api_url = api_url+str_trust_name+'/'
+    print(str_api_url)
```
Comment (Akshay14469) [inline NHS.Automation/FrameworkPython/common_files/common_functions.py]: remove this statement

**Review by Akshay14469 — DISMISSED** [review]

**Review by Akshay14469 — APPROVED** [review]
Approved as suggested by girish.

**Pull request merged**